drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -186,7 +183,7 @@ def studio_view(self, context):
 
         js_templates = loader.load_unicode('/templates/html/js_templates.html')
         help_texts = {
-            field_name: self._(field.help)
+            field_name: self.ugettext(field.help)
```

Comment:
@itsjeyd You are right at that point. Xblocks Message catalog is not loading in xblock `studio_view` while it is perfectly working for `student_view.` 
I had figured out this problem recently and I am working on a [PR
](https://github.com/edx/edx-platform/pull/11987) , After this merge you can see the translations in xblocks `edit` view. 
Let me know If you have any other questions ? 
cc: @mattdrayer 
Translation